package-lock.json

```diff
@@ -248,6 +248,22 @@
         "@bufbuild/buf-win32-x64": "1.26.1"
       }
     },
+    "node_modules/@bufbuild/buf-darwin-arm64": {
+      "version": "1.26.1",
+      "resolved": "https://registry.npmjs.org/@bufbuild/buf-darwin-arm64/-/buf-darwin-arm64-1.26.1.tgz",
+      "integrity": "sha512-nmyWiT/59RFja0ZuXFxjNGoAMDPTApU66CZUUevkFVWbNB9nzeQDjx2vsJyACY64k5fTgZiaelSiyppwObQknw==",
+      "cpu": [
+        "arm64"
+      ],
+      "dev": true,
+      "optional": true,
+      "os": [
+        "darwin"
+      ],
+      "engines": {
+        "node": ">=12"
+      }
+    },
```

Comment:
this was unintentional, but it reminds me that i've run into this issue with the `buf` tool as well, which is used for processing schemas